### PR TITLE
chore:update dependencies @W-14106783

### PR DESCRIPTION
**Pull request by ravijayaramappa**
## Details
Routine dependencies update

## Does this pull request introduce a breaking change?

* ✅ No, it does not introduce a breaking change.

## Does this pull request introduce an observable change?

* ✅ No, it does not introduce an observable change.

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
     "printWidth": 100,
     "singleQuote": true,
-    "tabWidth": 4
+    "tabWidth": 4,
+    "trailingComma": "es5"
```
Comment (ravijayaramappa) [inline .prettierrc:5]: Many files changed with the new behavior. Can enable the default behavior of prettier@v3.0 in a separate PR.

File: package.json [hunk 2]
```diff
@@ -81,7 +81,7 @@
         "node": ">=10"
     },
     "volta": {
-        "node": "16.20.1",
+        "node": "16.19.0",
```
Comment (ravijayaramappa) [inline package.json:84]: Should revert this change.

**Review by divmain — APPROVED** [review]

**Review by nolanlawson — APPROVED** [review]
